results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+- **JupyterLab’s default—meaning without the addition of extensions—UI customization capabilities are limited.** In conversation with participants and in observing the issues they faced, we’ve determined additional UI customization features that would help them better use JupyterLab. Entirely new customization options include selecting from multiple syntax highlighting themes, being able to select major button colors (like the `--jp-brand-color` family), being able to select state colors (like the warning, error, and info color families), and to set a maximum width for documents in the main document area. Additions to existing customization options include more themes beyond light and dark modes, to be able to change all text sizes in JupyterLab, to be able to select text sizes directly rather than in opaque increments (ie. to set font sizes to a point, pixel, or percent value rather than selecting menu items with no feedback), and to assign other or additional indicators for the status of running or freshly completed code cells.
+
+#### Different options for feedback on cell status 
+
+**Current default JupyterLab feedback for notebook cell status is exclusively visual**; the cell prompt and favicon changes rely on users being able to see the icons, especially since these changes are not surfaced to the user unless their focus happens to already be on those icons when they change. This makes knowing the status of a cell difficult to impossible for users who cannot look at the screen in full. Quick to run code cells with a cell output—like the majority of cells run during these tests— tend to make this a trivial issue, it marks a drastic experience difference for sighted and non-sighted users. Fully sighted users can quickly verify the status of a cell in two different places, go about other work while waiting, and more easily verify when a cell has completed running. 
+
+Participants who could not easily see or could not see the staus indicators at all (since they are small and relatively low contrast) relied on prior experience with notebooks, gut instinct about how long that section of code usually takes to run, or the appearance of a cell output to check a cell’s status. Having at least one non-visual option (like an audio indicator) or a visual option that would represent itself in a timely manner for assistive tech (like a notification) could provide users with the support they want. Making these options customizable would be important as well.
```

Comment:
```suggestion
Participants who could not easily see or could not see the status indicators at all (since they are small and relatively low contrast) relied on prior experience with notebooks, gut instinct about how long that section of code usually takes to run, or the appearance of a cell output to check a cell’s status. Having at least one non-visual option (like an audio indicator) or a visual option that would represent itself promptly for assistive tech (like a notification) could provide users the support they want. Making these options customizable would be necessary as well.
```